docs/install/installation.rst

```diff
@@ -17,31 +19,36 @@ Prerequisites
 
 - A C++ 17 compiler such as GCC 7 or Clang 5.
 
-- The AMD ROCm software stack. See the `ROCm installation instructions <https://rocm.docs.amd.com/projects/install-on-linux/en/latest/index.html>`_.
+- The AMD ROCm software stack. See the `ROCm installation instructions <rocm-install-on-linux:index>`.
+
+- Install the required packages according to the OS:
 
-- Packages as per the OS.
+.. tab-set::
 
-  - For Ubuntu 18.04 and Ubuntu 20.04:
+  .. tab-item:: Ubuntu 18.04 and Ubuntu 20.04
+    :sync: ubuntu
 
-  .. code:: shell
+    .. code-block:: shell
 
-    apt install gcc g++ make cmake libelf-dev libdw-dev
+      apt install gcc g++ make cmake libelf-dev libdw-dev
 
-  - For CentOS 8.1 and RHEL 8.1:
+  .. tab-item:: CentOS 8.1 and RHEL 8.1
```

Comment:
We probably can add RHEL 9 in the mix here.

So I think this is valid for RHEL 8 and RHEL 9. CentOS 8 (I think) still around as well.

Reply:
Updated.